### PR TITLE
Update EIP-1: Remove `requires` header

### PR DESCRIPTION
**Pull request by Pandapip1**
Removes the requires header in favor of it being automatically parsed from the links.

### CODE REVIEW

**Review by SamWilsn — COMMENTED** [review]
How would you feel about dropping the `requires` header entirely, treating all links as normative, and generating a machine readable bibliography while rendering from the markdown representations we allow?

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -186,11 +186,43 @@ The `category` header specifies the EIP's category. This is required for standar
...
+```yaml
+eip: example0
+...
+bib:
+  - id: EIP-example1
+    normative: false
+  - id: ERC-example2
+    normative: true
+  - id: 10.7717/peerj-cs.214
+    normative: false
```
[the remark this answers is not in the document]
Comment (Pandapip1) [reply EIPS/eip-1.md:224]: Yea, I'll change that

**Comment by Pandapip1** [comment]
> How would you feel about dropping the requires header entirely, treating all links as normative, and generating a machine readable bibliography while rendering from the markdown representations we allow?

So basically remove the requires header and automatically cite all references?

**Comment by SamWilsn** [comment]
> So basically remove the requires header and automatically cite all references?

Yes, exactly. Parse the shorthands defined in EIP-1 from the proposal being rendered and generate a true bibliography.

**Review by SamWilsn — COMMENTED** [review]
:+1: from me! I'll put it on the agenda for next EIPIP.

**Comment by SamWilsn** [comment]
From @xinbenlv, roughly:

> Generating the bibliography isn't blocked on removing the `requires:` header, and there isn't enough justification to remove the header at this time.

**Comment by Pandapip1** [comment]
Would there be any justification from keeping the header? I would be okay with grandfathering old EIPs.

**Review by Scamreno — COMMENTED** [review]
#7478

**Review by Scamreno — COMMENTED** [review]
#7478

**Review by Scamreno — COMMENTED** [review]
I just hope this finally be resolved

**Review by Scamreno — COMMENTED** [review]
-1.md

**Pull request closed without merging**